woocommerce/woocommerce-admin-test-helper
# Add loading and fix bug where the same keyword does not work

This PR adds a loading state and fixes the bug where entering the same keyword twice gets ignored.

### Screenshots

![Screen Shot 2021-03-27 at 10 08 20 AM](https://user-images.githubusercontent.com/4723145/112728426-7ade2a80-8ee4-11eb-9bca-880acf0fc227.jpg)

### Test Instructions

Loading state - Click "Search Option" button and you should see the "Loading..." text (screenshot attached)
